tests_integration/test_package.py

```diff
@@ -188,6 +196,19 @@ def test_package_with_native_libraries(self, directory_for_test, runner):
         assert result.exit_code == 1
         assert "at https://support.anaconda.com/" in result.output
 
+    @pytest.mark.integration
+    def test_incorrect_input(self, runner):
+        try:
+            runner.invoke_with_connection(
+                ["snowpark", "package", "create", "this is incorrect"]
+            )
+            raise "InvalidRequirement exception should be raised"
```

Comment:
Why we are rising this exception? Why not use `with pytest.rises`?

Reply:
Ah, I got some blackout and completely forgot. Thanks for catching!
Fixed